crates/bevy_ui/src/layout/debug.rs

```diff
@@ -0,0 +1,79 @@
+use crate::UiSurface;
+use bevy_ecs::prelude::Entity;
+use bevy_utils::HashMap;
+use taffy::prelude::Node;
+use taffy::tree::LayoutTree;
+
+pub fn print_ui_layout_tree(ui_surface: &UiSurface) {
```

Comment:
Basic docs please.

Reply:
Actually: why isn't this just a `Display` impl?

Reply:
I'm not sure what the conventions about `Display` impls are. This doesn't print everything known by `UiSurface`, only the layout tree.